problem5/calculate.c

```diff
@@ -37,17 +37,9 @@ int total(int numbers[], int length) {
 
 bool areStringsEqual(const char* first, const char* second) {
 
-bool result = strcmp(first, second);
+int result = strcmp(first, second);
 
-if (result == true) {
-    printf("no");
-}
-
-if (result == false) {
-    printf("yes");
-}
-
-return result;
+return strcmp(first, second)==0;
```

Comment:
So here, instead of calling `strcmp` again, we want to convert the `result` variable (which is an `int`) to a `bool` value and return that. We can do an `if/else` to do that.